storage/src/journal/contiguous/variable.rs

```diff
@@ -282,19 +282,19 @@ impl<E: Storage + Metrics, V: Codec + Send> Journal<E, V> {
     ///
     /// This count is NOT affected by pruning. The next appended item will receive this
     /// position as its value.
-    pub async fn size(&self) -> Result<u64, Error> {
-        Ok(self.size)
+    pub fn size(&self) -> u64 {
```

Comment:
I made the signature `pub async fn size(&self) -> Result<u64, Error>` to fulfill the `Contiguous` interface but we can wrap this in `Ok` and `async` in the `Contiguous for Journal` implementation below instead.